frontend/src/utils/utils.ts

```diff
@@ -63,6 +64,27 @@ export function partition<T>(
   );
 }
 
+/**
+ * @function excludeMetaTag
+ * Filter out a configured list of select metadata or tags from a COMS response
+ * @param {object} type either 'metadata' or 'tagset'
+ * @param {object} axiosResponse A resolved axios response from COMS
+ * @returns {object} The response data with select metadata/tags from a configured list removed
+ */
+export function excludeMetaTag(type: string, data: any){
+  // array of selected tags/metadata (keys) to hide from UI
+  const excludeArray = new ConfigService().getConfig().exclude?.[type].split(',') ?? [];
```

Comment:
Need to do a `trim` after the split as we are not guaranteed that environment variables are well formatted - this will at least handle any errant whitespaces users may put into the environment variable.
```ts
// Array of specific values to mask
const excludeArray = new ConfigService().getConfig().exclude?.[type].split(',').map(s => s.trim()) ?? [];
```